lib/compiler/passes/generate-js.js

```diff
@@ -1008,7 +1044,7 @@ function generateJS(ast, options) {
       "            : classEscape(part))",
       "        );",
       "",
-      "        return \"[\" + (expectation.inverted ? \"^\" : \"\") + escapedParts.join(\"\") + \"]\";",
+      "        return \"[\" + (expectation.inverted ? \"^\" : \"\") + escapedParts.join(\"\") + \"]\" + (expectation.unicode ? \"u\" : \"\");",
```

Comment:
This will output expectations like "expected [^a-z]u", which might not be enough for casual users.  Watch for confusion in the userbase and find something better if needed.